### DISCUSSION TITLE
Windows Package Manager v1.1.12701

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#1518] [Releases]
This release is the first development build after the Windows Package Manager 1.1 release candidate build for Windows 10 (1809+).

Experimental features have been enabled in this release. This build will be released to Windows Insider Dev builds, and Windows Package Manager Insiders.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.1.12701'>Windows Package Manager v1.1.12701</a>.</em>

**Comment by vedantmgoyal9** [comment]
Does this include validation of v1.1 schema?

Edit: Saw the reply on the discussion of the previous pre-release just now.

**Comment by spragginsdesigns** [comment]
I have been running Windows 11 Beta for over a month now and am in the windows insider program, will I automatically get this? @denelon

**Reply by denelon** [reply]
"Windows Insider Beta" will not get this update. This is targeting "Windows Insider Dev" and the Windows Package Manager Insiders running Windows 10. We will be using "Windows Insider Beta" for release candidates. You should have gotten the previous release built to target Windows 11 (functionally equivalent to https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653 targeting Windows 10) on Windows 11 Insider Beta.

**Reply by jz999999** [reply]
how to use winget through ssh?

**Reply by jedieaston** [reply]
You can't at the moment, see https://github.com/microsoft/winget-cli/issues/1474.

**Reply by spragginsdesigns** [reply]
> 

Thank you for the update. I have been using Oh-My-Posh just like I use Oh My Zsh in Kali Linux 2021.3. With PSGallery, auto suggestions etc., good lord Windows Terminal Preview is a gift from Microsoft, the tireless work of the community and God. 👏👏

**Comment by Rafa-VP** [comment]
Does this can run with Psexec or another remote command?
